pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -54,14 +55,15 @@ func NewConfigObserver(
 				configInformers.Config().V1().Ingresses().Informer().HasSynced,
 				kubeInformersForEtcdNamespace.Core().V1().Endpoints().Informer().HasSynced,
 				kubeInformers.Core().V1().Secrets().Informer().HasSynced,
+				kubeInformersForEtcdNamespace.Core().V1().ConfigMaps().Informer().HasSynced,
 			},
 		},
 		[]factory.Informer{operatorConfigInformers.Operator().V1().OpenShiftAPIServers().Informer()},
 		images.ObserveInternalRegistryHostname,
```

Comment:
add the configmap informer for the etcd namespace to the list of informers one line up

Reply:
https://github.com/openshift/cluster-openshift-apiserver-operator/pull/364#discussion_r427378029

looks like library-go does this automatically? if so the only informers i'd expect to pass to the constructor are those which aren't already in the listers

Or I totally misunderstood...